modules/customerassignmentedit.php

```diff
@@ -37,7 +37,7 @@
 
 if ($_GET['action'] == 'suspend') {
     $LMS->SuspendAssignment($_GET['id'], $_GET['suspend']);
-    $SESSION->redirect('?'.$SESSION->get('backto'));
+    $SESSION->redirect('?'.$SESSION->get('backto', true));
```

Comment:
Zobacz jak to jest, gdzie indziej robione.
Zwracam uwagę, że obecnie w większości miejsc zapamiętujemy `backto` globalnie, a tylko w wybranych **per zakładka** przeglądarki www.
Sztywna zmiana z użycia globalnego na zakładkowe spowoduje, że gdy będą następować wejścia do edycji zobowiązania z kodu obsługującego tylko użycie globalne to powrót po zakończeniu edycji może nie zadziałać sensownie.

Reply:
Akurat do tego miejsca można przejść z UI tylko i wyłącznie z customerinfo.php a tam jest zapamiętywanie sesji zrobione na dwa sposoby. Dodawanie warunków sprawi, że kod obsługi sesji się skomplikuje.

Chyba wolę spróbować hurtem zmienić true, w obsłudze sesji zamiast wracać do tematu co jakiś czas.